build.sbt

```diff
@@ -23,17 +23,24 @@ ThisBuild / pomExtra := (
 
 publish / skip := true
 
+libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.8" % Test
+
 lazy val cross = crossProject(JSPlatform, JVMPlatform)
   .crossType(CrossType.Pure)
   .in(file("."))
   .settings(
-    name := "minitime"
+    name := "minitime",
+    libraryDependencies += "org.scalatest" %%% "scalatest" % "3.0.8" % Test
```

Comment:
Scala.js dependency should be put in `.jsSettings()`

Reply:
What I expect is `scalatest` will run on JVM and Javascript. AFAIK, `%%%` is used to indicate libraries will be used for cross platform. Maybe I am wrong at that? 

> If your cross compiled source depends on libraries, you may use %%% for both projects. It will automatically determine whether you are in a Scala/JVM or a Scala.js project. For example, if your code uses Scalatags, your project definitions look like this:

https://www.scala-js.org/doc/project/cross-build.html